File: sdkwork-ai-sdk-go/types/category_scores.go
package types

// Category scores
type CategoryScores struct {
	Hate float64 `json:"hate"`
	Sexual float64 `json:"sexual"`
	Violence float64 `json:"violence"`
	HateThreatening float64 `json:"hate/threatening"`
	SelfHarm float64 `json:"self-harm"`
	SexualMinors float64 `json:"sexual/minors"`
	ViolenceGraphic float64 `json:"violence/graphic"`
	IllegalAct float64 `json:"illegal-act"`
	IllegalActInstruction float64 `json:"illegal-act/instruction"`
}

File: sdkwork-ai-sdk-typescript/src/types/category-scores.ts
/** Category scores */
export interface CategoryScores {
  /** Score for hate category */
  hate?: number;
  /** Score for sexual category */
  sexual?: number;
  /** Score for violence category */
  violence?: number;
  /** Score for hate/threatening category */
  'hate/threatening'?: number;
  /** Score for self-harm category */
  'self-harm'?: number;
  /** Score for sexual/minors category */
  'sexual/minors'?: number;
  /** Score for violence/graphic category */
  'violence/graphic'?: number;
  /** Score for illegal-act category */
  'illegal-act'?: number;
  /** Score for illegal-act/instruction category */
  'illegal-act/instruction'?: number;
}
